### ISSUE TITLE
Scaling + simplification drops errors

### ISSUE THREAD

**Issue by Enchufa2** [r-quantities/units#214]
We have:

```r
set_quantities(1.1, m, 0.1) * set_quantities(1.1, m^-1, 0.1)
## 1.2(2) [1]
```

Simplification to unitless, and errors are preserved. Ok. But at least when scaling is needed too, this happens:

```r
set_quantities(1100, mm, 100) * set_quantities(1.1, m^-1, 0.1)
## 1.21 [1]
set_quantities(1100, mm, 100) / set_quantities(1.1, m, 0.1)
## 1 [1]
```

Errors are dropped, which means that some path in `units` is not calling `NextMethod`.

**Comment by Enchufa2** [comment]
This behaviour seems related, because again errors are dropped:

```r
set_quantities(1, 1/nm) * set_quantities(1, m) / set_quantities(1, mm)
## 1e+09 [1]
## Warning message:
## Incompatible methods ("Ops.units", "Ops.quantities") for "/"
```

**Comment by Enchufa2** [comment]
Fixed in r-quantities/quantities#3. Not the ideal solution, but it works. There is room for improvement if we export here at some point functions to simplify units, as we were discussing in #132.
